test/diagnostics/diagnostics_tests.jl

```diff
@@ -4,7 +4,7 @@ using ClimaLand.Diagnostics: @with_error
 
 @test isdefined(ClimaLand.Diagnostics, :compute_sw_albedo!)
 
-@test !hasmethod(
+@test hasmethod(
```

Comment:
I think this should be `!`. The method gets defined below by `@diagnostic_compute`

Reply:
The test fails with!.
Let's understand what is happening